libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.h

```diff
@@ -264,7 +264,7 @@ typedef void (^SFOAuthBrowserFlowCallbackBlock)(BOOL);
 /**
  The current state of any in-progress advanced authentication flow.
  */
-@property (nonatomic, readonly) SFOAuthAdvancedAuthState advancedAuthState;
+@property (nonatomic, readonly) SFOAuthAdvancedAuthState advancedAuthState SFSDK_DEPRECATED(11.0, 12.0, "Will be removed");
```

Comment:
I don't see any internal app usage of this, thinking it could be retired?

Reply:
👍